lib/ember-build.js

```diff
@@ -194,7 +195,9 @@ var EmberBuild = CoreObject.extend({
       srcFile:  this._name + '.prod.js',
       destFile: this._name + '.min.js',
       mangle:   true,
-      compress: true
+      compress: {
+        global_defs: { DEV_MODE: false }
```

Comment:
I'm not a fan of this mechanism for knowing what "mode" we are in in Ember. I suppose the main reason would be that it would completely break when used in an ember-cli app which doesn't use the `ember.min.js` file.

We should instead use the mechanisms we already have setup.  I'll try to provide more details in the Ember PR.